pyproject.toml

```diff
@@ -34,6 +34,10 @@ dependencies = [
   "numpy",
   "scikit-learn>=1,<2",
   "matplotlib",
+  "onnxconverter-common",
+  "skl2onnx",
+  # onnxruntime is required only for testing
```

Comment:
Let's remove it, we don't have any other test dependencies here, like pytest. I created an issue for this
https://github.com/snad-space/coniferest/issues/107

For now, please add it to github action workflow in the same places where we install pytest

Reply:
I made it in https://github.com/snad-space/coniferest/pull/108, so now we have a special section of `pyproject.toml` to add dev dependencies to